recipes/bilm/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bilm" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # there is no Github release - so this ugly link is needed
+  url: https://github.com/allenai/bilm-tf/archive/ebf52c6ec1012a3672247c2d14ff7bcad7fb812b.zip
+  sha256: 5722ebd46f4120823469e1c221d4123cc6a1bc33b7083243e0893066e6f5a0f1
+
+build:
+  noarch: python
```

Comment:
I think you have to remove the ``noarch`` and add a ``build`` section inside of the ``requirements`` and after that add
```
  build:
    - {{ compiler('c') }}
```